frontend/src/components/WofButton.vue

```diff
@@ -87,6 +87,21 @@ export default {
     }
   }
 
+  &--outline {
+    color: black;
+    background-color: transparent;
+    border: 1px solid @primary-dark-text-color;
+    &:hover {
```

Comment:
For keyboard navigation, should probably add this to other variants as well. When I created the buttons, I wasn't aware of css best practices yet.
```suggestion
    &:hover &:active {
```